FirebaseSessions/Tests/Unit/SettingsTests.swift

```diff
@@ -0,0 +1,212 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import FirebaseSessions
+
+class SettingsTests: XCTestCase {
+  // 2021-11-01 @ 00:00:00 (EST)
+  let date = Date(timeIntervalSince1970: 1_635_739_200)
+  let validSettings: [String: Any] = [
+    "cache_duration": 10,
+    "sessions_enabled": false,
+    "sampling_rate": 0.5,
+    "session_timeout": 10,
+  ]
+  let corruptedJSONString: String = "{{{{ non_key: non\"value {}"
+  let fileManager: FileManager = .default
+  var settingsFileManager: SettingsFileManager!
+  var settings: Settings!
+  var appInfo: MockApplicationInfo!
+
+  override func setUp() {
+    appInfo = MockApplicationInfo()
+    settingsFileManager = SettingsFileManager(fileManager: fileManager)
+    settings = Settings(fileManager: settingsFileManager, appInfo: appInfo)
+  }
+
+  func test_noCacheSaved_returnsDefaultSettings() {
+    assert(settings.isCacheExpired == true)
```

Comment:
nit– I would favor XCTAssert for unit test assertions.

Reply:
```suggestion
    XCTAssertTrue(settings.isCacheExpired)
```

Reply:
Done